showcase/korogenoest.md

```diff
@@ -0,0 +1,130 @@
+
+### PCR testing
+The laboratory of  SYNLAB Eesti carried out PCR testing. They captured Ct values and  particular mutation sites for all positively tested samples. The social security codes that uniquely identify a person in Estonia were shared with the Health Board. Health Board  together with the virology experts from the University of Tartu chose samples for sequencing based on various parameters. 
+
+In general, the samples were carefully selected to ensure representation of the Estonian demographic and geographic subgroups, with additional samples taken from outbreaks that occurred at hospitals, elderly homes or work places. The data at individual level was linked at the Health Board and sample IDs from SYNLAB Eesti lab and the matching metadata IDs were created. Both the Health Board and SYNLAB Eesti uploaded the data about the samples to an access restricted GitLab repository. The University of Tartu team received only pseudonymised data. 
+
+In addition to sequencing carried out in Estonia, samples were also sent to ECDC. The metadata for these samples followed the same pipeline as above. But the sequences were downloaded from dedicated ECDC websites and bioinformatically processed the same manner as the samples sequenced in Estonia.
+
+### Metadata cleaning and standardised
+The metadata was semi-automatically cleaned and automatically standardised by custom-developed Python scripts. The following metadata types were cleaned and standardised -  age, age group, county, region of country, symptom onset date, place of infection,  vaccine producer and dose  (all applies to data being available).
+
+
+## Sequencing and sequence analysis
+
+Since May 2021 the samples moved from SYNLAB Eesti and hospitals laboratories directly to the University of Tartu Genome Centre Core lab who carried out the sequencing using Illumina NextSeq sequencer. Before, the samples were manually picked out from all the positive tests at the University of Tatu. Based on the sample information and original location on the laboratory plates the automatic pipeline generated the 96-well plate layouts needed to carry out the sequencing.
+
+The sequencing was carried out using [Arctic3](https://www.protocols.io/view/covid-19-artic-v3-illumina-library-construction-an-j8nlke665l5r/v5) and later [Arctic4](https://www.protocols.io/view/covid-19-artic-v4-1-illumina-library-construction-j8nlk4b36g5r/v2)  libraries.  At times also [NebNext Arctic](https://www.protocols.io/view/nebnext-artic-protocols-collection-rm7vz3rxrgx1/v1T) protocols were tested out. 
```

Comment:
```suggestion
The sequencing was carried out using [Arctic3](https://dx.doi.org/10.17504/protocols.io.bibtkann) and later [Arctic4](https://dx.doi.org/10.17504/protocols.io.j8nlk4b36g5r/v2)  libraries.  At times also [NebNext Arctic](https://dx.doi.org/10.17504/protocols.io.bw2apgae) protocols were tested out. 
```
Substituted links with DOIs, please check that they match

Reply:
ok